lib/usd/translators/meshWriter_BlendShapes.cpp

```diff
@@ -804,12 +849,20 @@ MObject PxrUsdTranslators_MeshWriter::writeBlendShapeData(UsdGeomMesh& primSchem
                         CHECK_MSTATUS_AND_RETURN(stat, MObject::kNullObj);
                         // NOTE: (yliangsiew) Because a single weight can drive multiple targets, we
                         // have to put a numeric suffix in the target name.
```

Comment:
Similar here. These could maybe just move into the `else` blocks?

Reply:
Resolved in 522b6b0.